src/ComScoreAnalytics.ts

```diff
@@ -66,6 +70,22 @@ export class ComScoreAnalytics {
     return new ComScoreStreamingAnalytics(player, metadata, this.configuration);
   }
 
+  /**
+   * sets the userContent to granted. Use after the ComScoreAnalytics object has been started
+   */
+  public static userConsentGranted() {
+    ns_.comScore.setLabels({ cs_ucfr: '1' });
+    ns_.comScore.hidden();
```

Comment:
What does `hidden` mean?

Reply:
its an method on the ComScore API that needs to be called after changing labels. I am just following what is in the documentation

Reply:
👍